content/logs/_index.md

```diff
@@ -20,6 +20,10 @@ logs_enabled: true
 
 The Datadog agent sends its logs to Datadog over TLS-encrypted TCP. This requires outbound communication over port `10516`.
 
+## Using a Proxy for Logs
+
+This is unavailable at this time using the `proxy` setting in the `datadog.yaml` configuration file. 
```

Comment:
The log agent does not presently respect the the `proxy` setting in the `datadog.yaml` configuration file.  This feature will be available in a future release.